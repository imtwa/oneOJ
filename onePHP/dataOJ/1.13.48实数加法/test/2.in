214.050041408
788.876404101