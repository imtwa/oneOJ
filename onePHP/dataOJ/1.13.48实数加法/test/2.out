1002.926445509
